pkg/status/status_test.go

```diff
@@ -248,22 +249,30 @@ func TestProbeLifecycle(t *testing.T) {
 		<-cancelled
 	}()
 
+	backends := Backends{
```

Comment:
maybe extract this to a helper?

Reply:
Yeah I debated and I thought the host, hash, visibillity vary enough and the struct was small enough that I didn't feel it necessary.